pkg/extension/workspace/example/runtime/configdir/testdata/workspace4/.getenvoy/extension/examples/default/cds.yaml

```diff
@@ -1,17 +0,0 @@
-version_info: v0
```

Comment:
I guess we could also delete `pkg/extension/workspace/example/runtime/configdir/testdata/workspace4/expected/getenvoy_extension_run/lds.tmpl.yaml` and crd.yaml?

Reply:
will try and delete if possible. sounds good!